src/interface/SpellLink.tsx

```diff
@@ -7,7 +8,7 @@ import SpellIcon from './SpellIcon';
 import useSpellInfo from './useSpellInfo';
 
 interface Props extends Omit<React.HTMLAttributes<HTMLAnchorElement>, 'id'> {
-  id: number;
+  id: number | Spell;
```

Comment:
An id can be a full Spell object? That doesn't seem correct

Reply:
In a lot of places, `SpellLink` is used with the entire `Spell` object (in fact, about 48 places). Even the comment mentions it 😄 I could go through all of them specify the `.id` and this could go away, if that's better.